tests/examples/wibox/widget/template/clone1.lua

```diff
@@ -0,0 +1,149 @@
+--DOC_GEN_IMAGE --DOC_NO_USAGE
```

Comment:
This example is quite interesting.

It's actually a basic implementation of the taglist/tasklist widgets with the new template. We only need a way to automatically attach a `self.widget_template:clone()` to each item on the tag/task list and assign them predefined `:set_property` and `:bind_property` to make it a perfect replacement.